packages/docsearch-react/src/utils/getAttributeValueByPath.ts

```diff
@@ -0,0 +1,6 @@
+export function getAttributeValueByPath<TRecord>(
+  record: TRecord,
+  path: string[]
+): any {
```

Comment:
you actually can type this if path is a readonly array, but not worth the hassle (like https://dev.to/pouja/making-lodash-function-get-type-safe-313o, but that's with a string, although the main part is the same)